internal/envoy/version.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package envoy
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/tetratelabs/getenvoy/internal/globals"
+	"github.com/tetratelabs/getenvoy/internal/version"
+)
+
+var (
+	versionVarName      = "$ENVOY_VERSION"
+	wdVersionFileName   = filepath.Join("$PWD", ".envoy-version")
+	homeVersionFileName = filepath.Join("$GETENVOY_HOME", "version")
```

Comment:
I was confused by these names so I would suggest 
```suggestion
	versionSourceVersionVar      = "$ENVOY_VERSION"
	versionSourceWDVersionFile   = filepath.Join("$PWD", ".envoy-version")
	versionSourceHomeVersionFile = filepath.Join("$GETENVOY_HOME", "version")
```

Reply:
how about using the word default instead of source? I find the word source awkward as it feels like source code..

Reply:
sounds better to me 😄